src/azure.js

```diff
@@ -3,7 +3,7 @@ var api         = require('./v1');
 
 api.declare({
   method:     'get',
-  route:      '/azure/:account/table/:table/read-write',
+  route:      '/azure/:account/table/:table',
```

Comment:
So we can change this to: `/azure/:account/table/:table/:level` without breaking the REST API, right?

We break the definition of how `auth.azureTableSAS` works in future versions of the client libraries.
So if we want to do the pretty thing and built least possible legacy, what we do is we break the client libraries, but not the REST API. Hence, older client libraries will still work, they just don't have the parameter.

Given that this is a method used in a very few places I don't think this is a problem. As long as we upgrade the node client to version 2, after we update the reference docs this should be fine. I doubt anyone using the python or golang clients are using `azureTableSAS`, but sure we should bump those too...

Note: I guess in general this might be a little bad practice... but given that this method is used few places, I think we should welcome the chance to avoid legacy in the name of backward compatibility.